src/dataflow-types/src/types/sinks.proto

```diff
@@ -17,6 +17,14 @@ import "repr/src/relation_and_scalar.proto";
 
 package mz_dataflow_types.types.sinks;
 
+message ProtoSinkConnector {
+    oneof kind {
+        ProtoKafkaSinkConnector kafka = 1;
+        google.protobuf.Empty tail = 2;
```

Comment:
What do you mean by that?

Reply:
as in, `TailConnector` is not really `()`, as someone may adds fields to it in the future. my understand is we are using `google.protobuf.Empty` because we cant do:
```
message TailConnector {
}
```
right?

Reply:
Actually, [that's exactly the definition](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/empty.proto#L51) of `google.protobuf.Empty`! `google.protobuf.Empty` exists [mostly so that](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Empty) we don't have to keep defining empty messages everywhere (since they are all the same anyway).